testsuite/forge-cli/src/main.rs

```diff
@@ -397,7 +397,7 @@ fn get_changelog(prev_commit: Option<&String>, upstream_commit: &str) -> String
 
 fn get_test_suite(suite_name: &str, duration: Duration) -> Result<ForgeConfig<'static>> {
     match suite_name {
-        "land_blocking" => Ok(land_blocking_test_suite(duration)),
+        "land_blocking" => single_test_suite("three_region_sim_graceful_overload"),
```

Comment:
Just a note to revert this before landing.

Reply:
@ibalajiarun typo :)

Reply:
I wasted an hour :(